Signal/translations/en.lproj/Localizable.strings

```diff
@@ -529,6 +529,15 @@
 /* message footer while attachment is uploading */
 "MESSAGE_STATUS_UPLOADING" = "Uploading...";
 
+/* Indicates that this 1:1 conversation has been blocked. */
+"MESSAGES_VIEW_CONTACT_BLOCKED" = "This User is Blocked";
```

Comment:
Maybe "You Blocked this User" to be clear that the user isn't e.g. banned from the signal service.